### PR TITLE
Setup, Add content and style

### PR DESCRIPTION
**Pull request by Moses-chibuike**
- Add linter
- Add content and style
- Add documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,53 @@
...
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (jerryowusu) [inline README.md]: Your readme is missing certain key/important sections of the [Microverse Readme template](https://github.com/microverseinc/readme-template) like the `Tech Stack`, `Key Features`, `Future Features`, as well as most parts of the Getting Started section, etc. Note that it is a requirement to produce all sections of the readme template in this and all other subsequent projects you do in Microverse here. Kindly add them and make sure to customized to match your project credentials. :pray:

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @Moses-chibuike , 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, there are few changes you need to make in order to move on to the next level :sunglasses:

this close2

## Highlights
All linters are passing :heavy_check_mark: 
Used github flow :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MeqdamAlqudah — APPROVED** [review]
Hi @Moses-chibuike , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

image

## Highlights 
:+1: Readme file
:+1: no linter errors

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Moses-chibuike) [reply README.md]: Thanks @jerryowusu for the review, this was very clear.

**Comment by Moses-chibuike** [comment]
That was swift @MeqdamAlqudah, thanks.

**Pull request merged**